lib/Mojo/Util.pm

```diff
@@ -23,6 +23,12 @@ use Unicode::Normalize ();
 # Check for monotonic clock support
 use constant MONOTONIC => eval { !!Time::HiRes::clock_gettime(Time::HiRes::CLOCK_MONOTONIC()) };
 
+# Speed up @{ $foo // [] } and similar by avoiding having to construct
+# a new empty arrayref each time
+use constant _EMPTY_ARRAY => [];
+# Make sure nobody accidentally modifies the referenced array
+BEGIN { Internals::SvREADONLY(@{+_EMPTY_ARRAY}, 1) }
```

Comment:
Did that really run through perltidy?

Reply:
The only thing perltidy did was add an empty line after the `use constant`, which seems silly. I've instead moved both the comment lines together before the block, and joined the with a `, and`.  Breaking the line after the comma seemed more sensible than forcing the first line to the full 120 characters specified in `.perltidyrc`.

Reply:
If you want to start an argument about our formatting rules here, i have to block this PR until that is resolved.

Reply:
Not really. I've split it back into to separate blocks now, with an empty line between, per perltidy.